### PR TITLE
Allow filtermodel to work for plugin models also

### PR DESCRIPTION
**Pull request by zachmullen**
@mgrauer this was a silly oversight on my part, PTAL.

### CODE REVIEW

File: plugins/jobs/server/job_rest.py [hunk 1]
```diff
@@ -48,54 +57,26 @@ def listJobs(self, params):
             user = self.model('user').load(
                 params['userId'], user=currentUser, level=AccessType.READ)
 
-        jobs = self.model('job', 'jobs').list(
+        return list(self.model('job', 'jobs').list(
```
Comment (mgrauer) [inline plugins/jobs/server/job_rest.py]: any reason you are preferring `list` over `[]`?

**Comment by mgrauer** [comment]
I'm not impressed, I've seen much sillier oversights.

LGTM.

File: plugins/jobs/server/job_rest.py (hunk 1 above)
Comment (brianhelba) [inline plugins/jobs/server/job_rest.py]: @mgrauer I'm not sure whether you mean a list comprehension or a list literal. 
I don't see how either is really appropriate here, the constructor seems correct to me.

**Comment by brianhelba** [comment]
LGTM.

Note that there is [a ``.. deprecated::`` block](http://sphinx-doc.org/markup/para.html#directive-deprecated), providing pretty formatting.

File: plugins/jobs/server/job_rest.py (hunk 1 above)
Comment (mgrauer) [inline plugins/jobs/server/job_rest.py]: I didn't read it carefully enough, thanks @brianhelba .

**Pull request merged**